code/modules/antagonists/abductor/abductor.dm

```diff
@@ -179,6 +179,12 @@
 	name = "Abductee"
 	roundend_category = "abductees"
 	antagpanel_category = "Abductee"
+	var/custom_objective
+
+/datum/antagonist/abductee/New(custom_objective)
+	. = ..()
+	if(custom_objective)
+		src.custom_objective = custom_objective
```

Comment:
Crash if custom objective is null rather than safely handle it in a way that causes future bugs without a stack trace

Reply:
But if `custom_objective` is null... it'll just do a random objective instead?

Reply:
Ah yea, I see. This is the antagonist and not an objective